central/platform/reprocessor/reprocessor.go

```diff
@@ -0,0 +1,12 @@
+package reprocessor
+
+// PlatformReprocessor reprocesses alerts and deployments to mark those that are platform components
+//
+//go:generate mockgen-wrapper
+type PlatformReprocessor interface {
```

Comment:
Is the reason we need this because the existing reprocessor does not look at alerts?  My first thought was why couldn't we just use the existing reprocessor for this, but I assume alerts is why.

Reply:
Yea the existing reprocessor does not look at alerts. It does reprocess deployments where it sends a reprocess message to sensor. And I think sensor only responds with deployments that have changed. So all deployments may not be reprocessed either. That is why I thought a new startup task would be better.